book/src/proposals/booting-from-snapshot.md

```diff
@@ -0,0 +1,21 @@
+# Booting From a Snapshot
+
+## Problem
+
+A validator booting from a snapshot should know that they are caught up to the cluster before voting.
+
+## Solution
+
+A booting validator needs to have some set of trusted validators whose votes it will rely upon to ensure that it has caught up to the cluster and has a valid snapshot. With that set, the validator can
+
+1. Get a snapshot to boot from.
+
+2. Send canary transactions with its local recent blockhash to test if they get onto the ledger.
```

Comment:
Let's formalize what it means to "get onto the ledger". @aeyakovenko implied this means seeing the tx in a root, but this doesn't seem to work if there are several valdiators booting up with > 33% of the stake.

@aeyakovenko suggested the above problem might be fixed if we make the test tx a vote, but once we fix https://github.com/solana-labs/solana/issues/5850, have the a validator vote once will not be sufficient to have the other validators pass threshold.

What if we just need to see that your set of trusteed validators voted on a block contaiining your text tx?

Reply:
My intention was that 4) covers that, will clarify.